.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = git://github.com/davepacheco/javascriptlint.git
+	url = git://github.com/joyent/javascriptlint.git
```

Comment:
Could theoretically switch these to https URLs as well?

Reply:
Oh, I guess `git://` accidentally worked thanked to our old friend the automation key.